### ISSUE TITLE
New Release

### ISSUE THREAD

**Issue by adamduren** [aldjemy/aldjemy#105]
According to PyPi there hasn't been a release since 2019-10-30 and there have been some commits since that address issues such as support for ArrayField. 

Are there plans to release a new version? Currently I'm installing based on a github commit reference.

**Comment by ryanhiebert** [comment]
over in #100 I was planning to make a release. I think it just fell off my radar, sorry about that. Would you prefer I release current master as 0.11, or go ahead and merge #100 (dropping support for some unsupported versions of Django and Python) and release 1.0?

**Comment by ryanhiebert** [comment]
I'm thinking to go ahead and merge the breaking changes around dropping those versions and release 1.0 right away, unless you tell me you'd prefer otherwise.

**Comment by ryanhiebert** [comment]
I'm finding myself quite busy today, I'm going to have to get to this next week. Sorry for the delay.

**Comment by ryanhiebert** [comment]
Release 1.0.0 is on PyPI. Sorry it took me so long.

**Comment by SaulTigh** [comment]
@ryanhiebert is there any known changelog for 1.0.0 ?
as for now changelog has been updated 2 years ago :(

**Comment by ryanhiebert** [comment]
I haven't been maintaining a changelog, unfortunately. I didn't realize there was previously one. The closest thing we have at the moment is the commit history. https://github.com/aldjemy/aldjemy/commits/1.0.0 Most of the changes were housekeeping, but there were two changes that added or improved functionality: #80, #101.

There were no breaking changes in 1.0. We dropped support for a bunch of versions of Django and Python from our build matrix, but I didn't actually remove any of the backward compatibility code. Because of that, I'm planning to make the next release 2.0, since we have merged some breaking changes since the 1.0 release. I don't have a timeframe on when 2.0 will be released. There are no interesting new features yet, so for the moment I'm content with what is currently released.
